src/main/scala/mesosphere/marathon/api/v2/validation/AppValidation.scala

```diff
@@ -231,7 +231,7 @@ trait AppValidation {
         option.keys.each should matchRegex(OptionKeyRegex)
       }
       val validExternalInfo: Validator[ExternalVolumeInfo] = validator[ExternalVolumeInfo] { info =>
-        info.name is definedAnd(matchRegex(LabelRegex))
+        info.name is notEmpty
```

Comment:
I believe we need to at least validate that name isn't empty, which is why I changed this to `is notEmpty`.

Reply:
I wish we had a character set that needed to be allowed for this feature. I'm a tad nervous that surrounding spaces or other strange characters will find their way in and wreak havoc some how.

Reply:
We had this discussion with @jrx and the result was _we really don't want to make any assumptions as to which character are valid and interpreted by the driver in any way or not_. So, sure, preceding or trailing `emspace` or whatnot sounds like something we should warn the user about – then again we have no control over what the driver does with them, and we most probably shouldn't make any assumptions.